README.md

```diff
@@ -4,6 +4,15 @@ A project based learning activity for people who are getting started with Git an
 
 You can play the game at: https://githubschool.github.io/github-games/
 
+### Instructions for playing the game
+
+1. Press the space bar to begin.
+2. Use the up and down arrow keys to rotate the shape.
+3. Use the left and right arrow keys to position the shape.
+4. The goal is to create complete rows with no empty spaces.
+5. When completed, the rows will disappear.
```

Comment:
Maybe specify that up is (say) clockwise and down is correspondingly anticlockwise
Are points awarded for a complete row to encourage progression in the game?
Possibly also award points for each piece delivered, maybe with dropping a shape awarding more points?